pydra/engine/tests/test_boutiques.py

```diff
@@ -99,24 +94,19 @@ def test_boutiques_spec_2(data_tests_dir):
 )
 def test_boutiques_wf_1(maskfile, plugin, tmpdir, infile):
     """wf with one task that runs fsl.bet using BoshTask"""
-    wf = Workflow(name="wf", input_spec=["maskfile", "infile"])
-    wf.inputs.maskfile = maskfile
-    wf.inputs.infile = infile
-    wf.cache_dir = tmpdir
-
-    wf.add(
-        BoshTask(
-            name="bet",
-            zenodo_id="1482743",
-            infile=wf.lzin.infile,
-            maskfile=wf.lzin.maskfile,
+
+    def Workflow(maskfile, infile):
```

Comment:
If I understand correctly?

```suggestion
    @workflow.define
    def Workflow(maskfile, infile):
```